include/s3select.h

```diff
@@ -855,6 +876,14 @@ void push_json_from_clause::builder(s3select* self, const char* a, const char* b
   self->getAction()->json_from_clause = variable_key_path;
 }
 
+void push_limit_clause::builder(s3select* self, const char* a, const char* b) const
+{
+  std::string token(a, b);
+
+  self->getAction()->limit_op = true;
+  self->getAction()->limit = std::stoul(token);
```

Comment:
it needs to check whether it is possible for `std::stoul` to throw an exception (ERANGE or something like that)
that may cause a crash (non-handled exception)